checker/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html><html><head><title>WebIDL 2 Checker</title>
+<script src='../lib/webidl2.js'></script>
+<script type='text/javascript'>
```

Comment:
nit: text/javascript is not required since HTML5

Reply:
Removed.